src/modules/copyFiles.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+
+const copyFiles = (command, fromFileDirectory, destinationFileDirectory) => {
+  if (command === 'cp' && fromFileDirectory !== destinationFileDirectory) {
+    let data = '';
+
+    try {
+      data = fs.readFileSync(fromFileDirectory, 'utf-8');
+    } catch (error) {
+      global.console.log('Unable to read file, no such file of directory');
+    }
+
+    try {
+      fs.writeFileSync(destinationFileDirectory, data);
+      global.console.log('File successfully copied');
+    } catch (error) {
+      global.console.log('Unable to write file');
```

Comment:
As a suggestion, you can also add a code of error and then text to it.

Reply:
yep, it can be informative

Reply:
fixed